airbyte-integrations/connectors/source-linkedin-ads/metadata.yaml

```diff
@@ -48,6 +48,23 @@ data:
               - "ad_member_seniority_analytics"
               - "ad_member_region_analytics"
               - "ad_member_company_analytics"
+      2.0.0:
+        message: This upgrade changes primary key for *-analytics streams from pivotValues[array of strings] to string_of_pivot_values[string].
```

Comment:
This upgrade changes primary key for *-analytics streams from pivotValues[array of strings] to string_of_pivot_values[string] so that it is compatible with more destination types.

Reply:
updated